sdk-test/index.js

```diff
@@ -0,0 +1,7 @@
+import { initCasperClient, LOCAL_NODE_URL } from '@invariant-labs/cspr-sdk'
+
+const main = async () => {
+  initCasperClient(LOCAL_NODE_URL)
```

Comment:
The best way to test it would be to use an example from the docs here to validate if everything works, the script is already written. Also, we can use typescript here